msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -148,7 +148,7 @@ The following table provides the user types and what features each user can acce
 | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
 | Anonymous user | Not available | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an adaptive card are allowed. | Not available |
```

Comment:
Note: Anonymous users inherit the global default user-level app permission policy. Refer to the "Manage Apps" section to learn more.

Reply:
Add this on top of the table